CondFormats/BeamSpotObjects/interface/SimBeamSpotObjects.h

```diff
@@ -87,6 +87,13 @@ class SimBeamSpotObjects {
   double alpha() const { return fAlpha; }
   double timeOffset() const { return fTimeOffset; }
 
+  /// Method to check if the object corresponds to GaussEvtVtxGenerator parameters
+  bool isGaussian() const {
+    // Check for the presence of GaussEvtVtxGenerator-specific parameters
+    return ((fMeanX != 0.0 && fMeanY != 0.0 && fMeanZ != 0.0) ||  // either centroid is not 0,0,0
```

Comment:
The condition "centroid is not 0,0,0" should be
```suggestion
    return ((fMeanX != 0.0 || fMeanY != 0.0 || fMeanZ != 0.0) ||  // either centroid is not 0,0,0
```
Besides being picky, couldn't it be that one of the three coordinates gets set to "0.0" on purpose?